docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -64,6 +64,52 @@ The dashboard is no longer accessible, even with the link, until you make it sha
 
 The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
 
+## Email Sharing
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+
+Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the url.
+
+### Enable email sharing
+
+> **Note:** For Grafana Cloud, you will need to contact support to have the feature enabled.
+
+Add the `publicDashboardsEmailSharing` feature toggle to your `custom.ini` file.
+
+```
+[feature_toggles]
+publicDashboardsEmailSharing = true
+```
+
+### Inviting a viewer
+
+1. Click **Only specified people**.
+2. Enter the email you want to share the public dashboard with.
+3. Click **Invite**.
+4. They will receive an email with a one-time use link. This link must be used within **1 hour** or it will be considered expired. Once the link is used, it will give the viewer access to the public dashboard for **30 days**.
```

Comment:
How do you get to step 1 from the dashboard's normal view state? We're missing a couple preliminary steps here. 

```suggestion
1. Click **Only specified people**.
1. Enter the email you want to share the public dashboard with.
1. Click **Invite**.
1. The recipient(s) will receive an email with a one-time use link. This link must be used within **one hour** or it expires. Once the link is used, the viewer has access to the public dashboard for **30 days**.
```

Reply:
Right, I'll add that. The steps for other pubdash actions above have it.

I was wondering why the number list was all 1's in other places. What's the point of that?

Reply:
> Right, I'll add that. The steps for other pubdash actions above have it.
> 
> I was wondering why the number list was all 1's in other places. What's the point of that?

using the 1. method basically eliminates human error in numbering and allows you to really easily add in new steps anywhere in a task without having to re-number everything. basically it's like letting markdown do some formatting for you rather than doing it manually.